bin/spack2cmake.sh

```diff
@@ -0,0 +1,45 @@
+#! /usr/bin/env bash
```

Comment:
I think this should maybe go into `bluebrain/bin` to keep things a wee bit more separate from upstream.

Reply:
Good, thank you.